src/util/native_fs_utils.js

```diff
@@ -462,6 +462,28 @@ async function config_file_exists(fs_context, config_path, use_lstat) {
     return true;
 }
 
+/**
+ * delete bucket specific temp folder from bucket storage path, config.NSFS_TEMP_DIR_NAME_<bucket_id>
+ * @param {nb.NativeFSContext} fs_context 
+ * @param {string} bucket_temp_dir_path
+ */
+async function delete_bucket_temp_dirs(fs_context, bucket_temp_dir_path) {
+    try {
+        const exists = await is_path_exists(fs_context, bucket_temp_dir_path);
+        if (!exists) {
+            return;
+        }
+        const entries = await nb_native().fs.readdir(fs_context, bucket_temp_dir_path);
+        for (const entry of entries) {
+            const temp_dir_path = path.join(bucket_temp_dir_path, entry.name);
+            await nb_native().fs.rmdir(fs_context, temp_dir_path);
+        }
+        await nb_native().fs.rmdir(fs_context, bucket_temp_dir_path);
+    } catch (err) {
+        throw new RpcError('BUCKET_CLEANUP_FAILED');
```

Comment:
maybe change it to be more indicatie:

```suggestion
        throw new RpcError('DELETE_BUCKET_TEMP_DIRS_FAILED');
```